ui-particles-angular/package.json

```
@@ -40,6 +40,7 @@
     "highcharts": "^9.3.2",
     "highlight.js": "^11.4.0",
     "lodash": "^4.17.21",
+    "ngx-file-helpers": "7.0.0",
```

Is it possible to use the last version ? 

```suggestion
    "ngx-file-helpers": "8.1.1",
```

---

because 8.* is for ng 14 not 12 as in apim